### DISCUSSION TITLE
Vote to retire @joshzarrabi

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#87] [Builder Team]
Hey @paketo-buildpacks/builders-maintainers, I'd like to propose to move @joshzarrabi to an emeritus contributor position. Josh has made some great contributions, but is now less involved in the project. We'd like to make sure his contributions are recognized. He will receive an emeritus status recognition on https://github.com/paketo-buildpacks/community. He will then also be removed from his current position as an active contributor to the Builders team.

Can I get a 👍 / 👎 vote?

**Comment by ryanmoran** [comment]
With a super-majority vote to approve, @joshzarrabi will be retired from the Builders Contributors team.

**Comment by joshzarrabi** [comment]
Being a maintainer of this repo was the greatest honor of my life.
